### PR TITLE
Add a new module to compute the attendances into the employee's payslip

### PR DESCRIPTION
**Pull request by dcoopennet**
New module to compute the attendances into the employee's payslip to add time compensation and salary compensation based on the working schedule.

For more detail README.rst

### CODE REVIEW

**Comment by yvaucher** [comment]
Please see the few lint errors raised by travis: https://travis-ci.org/OCA/l10n-switzerland/jobs/122807594

**Comment by yvaucher** [comment]
Please rebase it on 8.0, CI tests have been fixed

File: l10n_ch_payroll_attendances/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
```
Comment (yvaucher) [inline l10n_ch_payroll_attendances/__openerp__.py:4]: See new simplyfied template for licence headers:

https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3

File: l10n_ch_payroll_attendances/data/salary_rules.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<odoo>
+<data>
```
Comment (yvaucher) [inline l10n_ch_payroll_attendances/data/salary_rules.xml]: If you use `<odoo>` tag please remove `<data>` tag

File: l10n_ch_payroll_attendances/data/salary_rules.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        if (payslip_attendance.salary_increase):
+            time_to_compensate += payslip_attendance.salary_increase * payslip_attendance.nb_hours
+result = (contract.hourly_rate_attendance * time_to_compensate)
+        </field>
+        <field name="quantity">1</field>
+        <field name="sequence">300</field>
+        <field name="category_id" ref="hr_payroll.ALW"/>
+    </record>
+</data>
+</odoo>
```
Comment (yvaucher) [inline l10n_ch_payroll_attendances/data/salary_rules.xml]: Missing EOL here.

File: l10n_ch_payroll_attendances/__init__.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yvaucher) [inline l10n_ch_payroll_attendances/__init__.py]: Headers can be dropped from `__init__.py` files as there is no code.

File: l10n_ch_payroll_attendances/README.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+.. image:: http://s28.postimg.org/bittrg17h/Screen_Shot_04_13_16_at_03_05_PM.png
+
+- This result is used to create allocation request if time compensation was set.
+- It is also used to add salary compensation if you correctly set the corresponding rule into the salary structur of the employee.
+
+
+Contributors
+------------
+
+* David Coninckx <dco@open-net.ch>
```
Comment (yvaucher) [inline l10n_ch_payroll_attendances/README.rst:74]: Please see the template, there are some missing informations:

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Comment by yvaucher** [comment]
Coverage decreased (-28.5%) to 62.155% 

Your module definitely needs some unit tests

In 9.0 I want to aim 90% of coverage

**Comment by yvaucher** [comment]
Is this module specific to switzerland localisation?

Couldn't it land in OCA/hr by the way ?

**Comment by dcoopennet** [comment]
This module is not specifiy to switzerland localisation.
BUT we already have 2 modules https://github.com/OCA/hr/pull/200 here in a PR.
And my last message on this PR is dated of the 31 March 2016. 
That's the reason why we decided to try on l10n-ch.
But if you (yvaucher) can merge on the branch hr, We can try PR on this branch instead of l10n_ch.

Thanks a lot for your quick and smart answers.

**Comment by open-net-sarl** [comment]
Hi,

Another part of the thinking was the idea to keep more freedom on the way to add compensation rules.

We know about CH, we will implement with CH customers, and it might not fit all....

**Comment by yvaucher** [comment]
Ok I see OCA/hr#200 needs some eyeballs

**Comment by dcoopennet** [comment]
@yvaucher Is there something else to fix ? Can you merge ?

**Comment by dcoopennet** [comment]
@yvaucher Is there a reason why this PR can't be merged ?
Thanks

**Comment by yvaucher** [comment]
@dcoopennet shouldn't this PR be closed in favor of  OCA/hr#200 ?

**Comment by yvaucher** [comment]
@dcoopennet my bad those are different modules. Still, I think it would be more suited in OCA/hr

**Comment by dcoopennet** [comment]
@yvaucher It would be more suited in OCA/hr of course. BUT can you please review this module if i PR on hr ?

**Comment by yvaucher** [comment]
@dcoopennet I'll do it as I did for OCA/hr#200 you can also reference this PR for history.

**Pull request closed without merging**